app/src/main/java/protect/card_locker/Utils.java

```diff
@@ -105,7 +106,12 @@ static public BarcodeValues parseSetBarcodeActivityResult(int requestCode, int r
 
             Bitmap bitmap;
             try {
-                bitmap = MediaStore.Images.Media.getBitmap(context.getContentResolver(), intent.getData());
+                if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
```

Comment:
Nitpick, but could you please switch this around, so that it's if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q)?

I just prefer to have the new, proper code on top and the fallback in the else, instead of the fallback for old Android versions as the main thing.